_local/scripts/build-git.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+GIT_ARCHIVE="${GIT_VERSION}.tar.gz"
+GIT_SRC_ROOT=$(dirname "${GIT_DIST_PATH}")
+GIT_URL=https://github.com/git/git/archive/refs/tags
+if [[ ${GIT_VERSION} =~ master ]]; then
+    GIT_URL=https://github.com/git/git/archive/refs/heads
+fi
+GIT_URL="${GIT_URL}/${GIT_ARCHIVE}"
+
+# git clone issues:
```

Comment:
Not sure what you mean by these comments. Are this errors that happen intermittently? Or are they issues that were fixed by the code below? Please clarify.

Reply:
The comments below `git clone issues:` were fixed by the code which follows it.

Reply:
Thank you for clarifying. :+1: